application/core/plugins/AuditLog/AuditLog.php

```diff
@@ -395,14 +409,52 @@ public function beforeTokenDelete()
                 $oAutoLog->uid = $oCurrentUser->uid;
                 $oAutoLog->entity = 'token';
                 $oAutoLog->action = 'delete';
-                $oAutoLog->entityid = $aValues['participant_id'];
+                $oAutoLog->entityid = $aValues['participant_id'];   // TODO: Should this be the token ID?
```

Comment:
? You right : participant_id is from https://github.com/LimeSurvey/LimeSurvey/blob/8c332cc638306875f965ccd347ef5e5573f27ed0/application/models/Participant.php#L22 not in token model.

Reply:
It's another issue : not related here.

Reply:
Created new issue: https://bugs.limesurvey.org/view.php?id=19325